net.studio08.xbplay.metainfo.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id type="desktop">net.studio08.xbplay</id>
+    <launchable type="desktop-id">net.studio08.xbplay.desktop</launchable>
+    <name>xbPlay: Remote Play for Xbox</name>
+    <summary>Remote play from your console or xCloud</summary>
+    <metadata_license>CC-BY-SA-4.0</metadata_license>
+    <project_license>Apache-2.0</project_license>
```

Comment:
Out of curiosity, where do I find source code?

Reply:
Still in a private repo at this point. Planning to make it public in the near future.

Reply:
So the license is a lie. Since there is no source code.

Reply:
To be fair, I don't think Apache-2.0 technically requires the source to be available.